qa/L0_grpc/test.sh

```diff
@@ -142,14 +138,6 @@ for i in \
             cat $CLIENT_LOG.${SUFFIX}
             RET=1
         fi
-    elif [ $SUFFIX == "ensemble_image_client" ]; then
```

Comment:
Filed a follow up ticket to check this out [DLIS-2316](https://jirasw.nvidia.com/browse/DLIS-2316)

Reply:
Since we plan to keep an example of using ensemble for image classification, I am inclined to keep the test for that as well. So just comment this part out so it is easier to re-enable the test after the ensemble example is updated.

Reply:
Done